src/Pattern/index.ts

```diff
@@ -1,2 +1,2 @@
 export { Pattern } from './Pattern';
-export type * from './types';
+export * from './types';
```

Comment:
this is unrelated and as far as i understand will bring back the extra dumb bundling object. Don't mix up this change, we will check up this during the TS prs